forms-flow-theme/scss/_forms.scss

```diff
@@ -116,18 +116,20 @@ select option:hover {
 .form-control-with-icon {
   position: relative;
     .search-box-input {
-      border-radius:  var(--radius-0100);
+      border-radius:  16px !important; //TBD: To be changed to rem
       border: 1px solid var(--ff-primary);
       background:  var(--ff-white);
       color: var(--ff-gray-400);
       font-size: var(--font-size-xs);
       font-weight: var(--font-weight-sm);
       line-height: 120%;
```

Comment:
line height also can be added as variable (var(--text-line-height)).

Reply:
Updated